src/pkg/packager/deploy.go

```diff
@@ -337,7 +336,7 @@ func (p *Packager) getUpdatedValueTemplate(component types.ZarfComponent) (value
 
 	// If no distro the zarf secret did not load properly
 	if err != nil || state.Distro == "" {
-		return values, fmt.Errorf(lang.ErrLoadState, err)
```

Comment:
Just curious, why not use the lang files error message?

Reply:
?